Invoke-TrimarcADChecks.ps1

```diff
@@ -1,507 +1,653 @@
-<#.  
-SCRIPTNAME: Invoke-TrimarcADChecks.ps1
-AUTHOR: Sean Metcalf
-COMPANY: Trimarc Security, LLC (Trimarc)
-WEBSITE: https://www.TrimarcSecurity.com
-
-Last Updated: 07/06/2020
-Version 1.2020.07.06.13
+<#
+.SYNOPSIS
+Performs AD Scan
 
+.DESCRIPTION
 This script is designed for a single AD forest and is not designed to capture all data for a multiple domain forest.
 Note that if this script is used for a single domain in a multi-domain AD forest, not all elements may be captured.
 
+.PARAMETER DomainName
+Forest Name of your AD
+
+.PARAMETER RootDir
+Location to save all output too
+
+.EXAMPLE
+PS>.\Invoke-TrimarcADChecks.ps1
```

Comment:
Since $DomainName is pre-defined with ad.vulndomain.corp, this example fails.

Reply:
I added another comment in the parameter section explaining my preferred method of fixing this issue.